Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerBehaviorTests.cs

```diff
@@ -0,0 +1,283 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using System.Collections;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    /// <summary>
+    /// Verify that pointers can be turned on and offvia FocusProvider.SetPointerBehavior
+    /// </summary>
+    public class PointerBehaviorTests : BasePlayModeTests
+    {
+        private class PointerStateContainer
+        {
+            public bool? LinePointerEnabled { get; set; }
+            public bool? SpherePointerEnabled { get; set; }
+            public bool? PokePointerEnabled { get; set; }
+            public bool? GazePointerEnabled { get; set; }
+            public bool? GGVPointerEnabled { get; set; }
+        }
+
+        private void EnsurePointerStates(Handedness h, PointerStateContainer c)
+        {
+            Action<IMixedRealityPointer, string, bool?> helper = (ptr, name, expected) =>
+            {
+                if (!expected.HasValue)
+                {
+                    Assert.Null(ptr, $"Expected {h} {name} to be null but it was not null");
+                }
+                else
+                {
+                    Assert.NotNull(ptr, $"Expected {name} to not be null, but it was null");
+                    Assert.AreEqual(expected.Value, ptr.IsInteractionEnabled,
+                    $"Incorrect state for {h} {name}.IsInteractionEnabled");
+                }
+
+            };
+            helper(PointerUtils.GetPointer<LinePointer>(h), "Line Pointer", c.LinePointerEnabled);
+            helper(PointerUtils.GetPointer<SpherePointer>(h), "Sphere Pointer", c.SpherePointerEnabled);
+            helper(PointerUtils.GetPointer<PokePointer>(h), "Poke Pointer", c.PokePointerEnabled);
+            helper(PointerUtils.GetPointer<GGVPointer>(h), "GGV Pointer", c.GGVPointerEnabled);
+            helper(CoreServices.InputSystem.GazeProvider.GazePointer, "Gaze Pointer", c.GazePointerEnabled);
+        }
+
+        /// <summary>
+        /// Tests that the gaze pointer can be turned on and off
+        /// </summary>
+        /// <returns></returns>
```

Comment:
>/// <returns></returns> [](start = 8, length = 23)

Delete this line

Reply:
(Actually, all empty returns statements - these will also fail CI, or should fail CI)

---
In reply to: [323503717](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5920#discussion_r323503717) [](ancestors = 323503717)